yarn.lock

```diff
@@ -3152,7 +3152,12 @@
     web-vitals "^3.3.2"
     wolfy87-eventemitter "^5.2.9"
 
-"@guardian/consent-management-platform@13.5.0", "@guardian/consent-management-platform@^13.5.0":
+"@guardian/consent-management-platform@13.6.1":
+  version "13.6.1"
+  resolved "https://registry.yarnpkg.com/@guardian/consent-management-platform/-/consent-management-platform-13.6.1.tgz#528ed9193b1bd96e9f84964a5a1a080de43e29c1"
+  integrity sha512-5KQ4dZrqIBKuJCSJ/TTWXZ4F0WXY9ecpMnfOfvNIU9tXF1LAeT+LNv7hfV8eC5qMJskj2yvZZUi7hlwWD+Nq/A==
+
+"@guardian/consent-management-platform@^13.5.0":
   version "13.5.0"
   resolved "https://registry.yarnpkg.com/@guardian/consent-management-platform/-/consent-management-platform-13.5.0.tgz#61a97c823f6b18f62a6517a3374f6e7bdfca121d"
   integrity sha512-9iz04lyonWAbUeZ7B+fNp0KbgAervsWDO3A4CE/3e+d876AQj0I3TwU7W4jpWxI4tthcA6UVk7Y6faaS8l+lFw==
```

Comment:
It looks like `guardian/commercial` still depends on the previous version so we may end up with both versions in use? Is that likely to cause problems? Should we be updating `guardian/commercial` at the same time to make sure we stay on a single version?

cc @guardian/commercial-dev